pkg/kube/container_oci.go

```diff
@@ -248,35 +249,51 @@ func (t *containerTranslator) configureResources() {
 }
 
 func (t *containerTranslator) configureProcess() error {
-	const (
-		execScript = "/.singularity.d/actions/exec"
-		runScript  = "/.singularity.d/actions/run"
-	)
+	cmd := t.cont.GetCommand()
+	args := t.cont.GetArgs()
+	cwd := t.cont.GetWorkingDir()
+
 	if t.cont.imgInfo.OciConfig != nil {
```

Comment:
We should make sure to document that if the image has an OCI configuration the environment will reflect that instead of the environment that was set by the singularity image build process

Reply:
you mean we may loose envs when bootstrapping from docker and having `%environment` section? hm, maybe I can add extra flag to identify images that were build from docker, not bootstrapped. Do you think that will be better, @ikaneshiro?

Reply:
Yes, I think one way is to check the embedded definition and see if there's any sections. If it's just a header for a build from an oci source, then I think you should be good to just use the oci config env

Reply:
Can be done easier in CRI, will update soon
Thanks for suggestion